generators/entity/templates/src/main/webapp/app/entities/infinite-scroll-template.ejs

```diff
@@ -29,7 +29,11 @@
         <%_ } else { _%>
             if (vm.currentSearch) {
```

Comment:
In the case of infinite scrolling, if the user typed something in the search box then scrolled down to load the next page, they would get page 2 of the search results for whatever they searched instead of what they were expecting.